core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -580,7 +582,7 @@ object ConsumerGroupCommand extends Logging {
             groupId,
             Option(consumerGroup.coordinator),
             unassignedPartitions.keySet.toSeq,
-            unassignedPartitions.map { case (tp, offset) => tp -> Some(offset.offset) },
+            unassignedPartitions.map { case (tp, offset) => tp -> getPartitionOffset(tp) },
```

Comment:
Same for this one.

Reply:
Same with the previous approach, it could be from:
**From**:
```
val rowsWithoutConsumer = if (unassignedPartitions.nonEmpty) {
          collectConsumerAssignment(
            groupId,
            Option(consumerGroup.coordinator),
            unassignedPartitions.keySet.toSeq,
            unassignedPartitions.map { case (tp, offset) => tp -> getPartitionOffset(tp) },
            Some(MISSING_COLUMN_VALUE),
            Some(MISSING_COLUMN_VALUE),
            Some(MISSING_COLUMN_VALUE)).toSeq
        } else
          Seq.empty
```
**To**:
```
val rowsWithoutConsumer = if (unassignedPartitions.nonEmpty) {
          collectConsumerAssignment(
            groupId,
            Option(consumerGroup.coordinator),
            unassignedPartitions.keySet.toSeq,
            getPartitionOffsets(unassignedPartitions.keySet.toSeq),
            Some(MISSING_COLUMN_VALUE),
            Some(MISSING_COLUMN_VALUE),
            Some(MISSING_COLUMN_VALUE)).toSeq
        } else
          Seq.empty
```